doc/source/user_guide/adaptive_templates.rst

```diff
@@ -342,82 +328,70 @@ The ``triangular_notch()`` function requires 4 arguments:
 
     Required arguments for the ``triangular_notch()`` function.
 
-A rotor notch can be defined using this function,
-so that the user does not need to calculate
-the coordinates for the notch entities.
+A rotor notch can be defined using this function, so that the user does not need to calculate the
+coordinates for the notch entities.
 
-To use the ``triangular_notch()`` function to create
-a triangular rotor notch region:
+To use the ``triangular_notch()`` function to create a triangular rotor notch region:
 
 .. code:: python
 
     notch = triangular_notch(
         rotor_radius, notch_angular_width, notch_centre_angle, notch_depth
     )
 
-where the arguments ``rotor_radius``, ``notch_angular_width``,
-``notch_centre_angle`` and ``notch_depth``
-must be calculated in the Adaptive Templates Script and
-specified.
+where the arguments ``rotor_radius``, ``notch_angular_width``, ``notch_centre_angle`` and
+``notch_depth`` must be calculated in the Adaptive Templates Script and specified.
 
-The notch region properties can then be defined and
-the region can be set in Motor-CAD,
-as described earlier in this guide.
+The notch region properties can then be defined and the region can be set in Motor-CAD, as described
+earlier in this guide.
 
-For a full Adaptive Templates example
-using the workflow described here,
-see :ref:`ref_BPM_Triangular_Rotor_Notches`.
+For a full Adaptive Templates example using the workflow described here, see
+:ref:`ref_BPM_Triangular_Rotor_Notches`.
 
-Details on the Geometry Shapes functions within ``ansys.motorcad.core.geometry_shapes``
-are available in the :ref:`ref_API_Reference` under :ref:`ref_geometry_shapes`.
+Details on the Geometry Shapes functions within ``ansys.motorcad.core.geometry_shapes`` are
+available in the :ref:`ref_API_Reference` under :ref:`ref_geometry_shapes`.
 
 Creating and modifying adaptive templates scripts
 *************************************************
```

Comment:
```suggestion
Create and modify adaptive templates scripts
*******************************************
```
Use simple verbs in headings and titles (rather than gerunds, which are "ing" forms).